README.md

```diff
@@ -65,12 +77,16 @@ Use the command `apos-code-upgrader lint` to run a linter scan of the A2 code. T
 
 ## Running the code upgrade
 
-1. Type `apos-code-upgrader upgrade` in the project root to have the tool actually convert code to A3's expected structure and syntax. There may be immediate messages printed to the console suggesting next steps.
+1. Type `apos-code-upgrader upgrade` in the project root to have the tool actually convert code to A3's expected structure and syntax where possible. There may be immediate messages printed to the console suggesting next steps.
 2. Run the linter command, `apos-code-upgrader lint`, to see any remaining changes that are detected and require manual conversion.
 3. Review the changes (before committing them) with `git status` and `git diff HEAD`. Even though files are moved and directories renamed, git will still be able to display line changes for most of them.
 
 ### Reset the changes before committing
 
-If you want to undo all the changes made by the tool for any reason, run `apos-code-upgrader reset` in the project root. You must do this *before committing the changes*. This does a simple hard reset with git and will let you start again after reviewing the changes if you desire.
+If you want to undo all the changes made by the tool for any reason, run `apos-code-upgrader reset` in the project root. You must do this *before committing the changes*. This does a simple hard reset of your local branch with git and will let you start again after reviewing the changes if you desire.
 
 As a last resort you can always switch back to the main git branch and create a new upgrade branch to start over. You did switch off the main branch at the start, didn't you? 🤓
+
+## Pre-alpha options
```

Comment:
This should be a third level heading before the reset bit.

Reply:
I don't think we need to specify that they are pre-alpha. Simply "Options." As it evolves out of alpha the documentation will change.